Week1/homework/app.js

```diff
@@ -1,11 +1,187 @@
+    },
+    memory_in_flesh: {
+      title: 'Memory in Flesh',
+      language: 'Arabic',
+      author: 'Ahlam Mostaganmi',
+      ImageAlt: 'book cover Memory in Flesh',
+    },
+    dust_of_promises: {
+      title: 'Dust of Promises',
+      language: 'Arabic',
+      author: 'Ahlam Mostaganmi',
+      ImageAlt: 'book cover Dust of Promises',
+    },
+    art_of_forgetting: {
+      title: 'Art of Forgetting',
+      language: 'Arabic',
+      author: 'Ahlam Mostaganmi',
+      ImageAlt: 'book cover Art of Forgetting',
+    },
+    black_suits_you: {
+      title: 'Black Suits You',
+      language: 'Arabic',
+      author: 'Ahlam Mostaganmi',
+      ImageAlt: 'book cover Black Suits You',
+    },
+    chaos_of_senses: {
+      title: 'Chaos Of Senses',
+      language: 'Arabic',
+      author: 'Ahlam Mostaganmi',
+      ImageAlt: 'book cover Black Suits You',
+    },
+    bed_hopper: {
+      title: 'Bed Hopper',
+      language: 'Arabic',
+      author: 'Ahlam Mostaganmi',
+      ImageAlt: 'book cover Bed Hopper',
+    },
+    game_of_thrones: {
+      title: 'Game of Thrones',
+      language: 'English',
+      author: 'George R.R. Martin',
+      ImageAlt: 'book cover Bed Hopper',
+    },
+  };
+
+  for (const key of Object.keys(books)) {
+    console.log(books[key]);
+  }
+
+  const booksPath = {
+    deep_work: './images/deep_work.jpg',
+    anna_frank: './images/anna_frank.jpg',
+    forty_rules: './images/forty_rules.jpg',
+    memory_in_flesh: './images/memory_in_flesh.jpg',
+    dust_of_promises: './images/dust_of_promises.jpg',
+    art_of_forgetting: './images/art_of_forgetting.jpg',
+    black_suits_you: './images/black_suits_you.jpg',
+    chaos_of_senses: './images/chaos_of_senses.jpg',
+    bed_hopper: './images/bed_hopper.jpg',
+    game_of_thrones: './images/game_of_thrones.jpg',
+  };
+
+  function elements() {
+    const root = document.getElementById('root');
+    const headerTitle = document.createElement('h1');
+    headerTitle.innerText = 'I have read these 10 books';
+    root.appendChild(headerTitle);
+    const ul = document.createElement('ul');
+    root.appendChild(ul);
+
+    for (const key of Object.keys(books)) {
+      const liElem = books[key];
+      const li = document.createElement('li');
+      ul.appendChild(li);
+      li.setAttribute('id', key);
+
+      const div = document.createElement('div');
+      li.appendChild(div);
+      div.setAttribute('class', 'item-container');
+      const imageContainer = document.createElement('div');
+      div.appendChild(imageContainer);
+      imageContainer.setAttribute('class', 'image_container');
+      const contentContainer = document.createElement('div');
+      contentContainer.setAttribute('class', 'content_container');
+      div.appendChild(contentContainer);
+      const bookImage = document.createElement('img');
+      bookImage.setAttribute('src', booksPath[key]);
+      bookImage.setAttribute('alt', liElem.imageAlt);
```

Comment:
The key should be `ImageAlt`, mind the capital `I`.

Reply:
Oops! I understood from Naming Convention it should be camelCase. Am I right? 
Thanks in advance! :)

Reply:
Take a look at line 73, `ImageAlt: 'book cover Deep Work',` you have capital `I` which should be lowercase ;)